docs/upgrade_oracle.md

```diff
@@ -0,0 +1,35 @@
+# kava-8 Oracle Update Guide
```

Comment:
Put these files (upgrade_oracle, validator, and wallet) inside a directory for `kava-8`. So, `docs/kava-8/update_oracle.md`, etc.

Reply:
Ok - addressed [here](https://github.com/Kava-Labs/kava/pull/1020/commits/ec60bf16c36544df57e84d4b30585f410b471e13)